test/contrib/elasticsearch/quantize_test.rb

```diff
@@ -9,7 +9,7 @@ def test_id
     assert_equal('/my/thing/?/', Datadog::Contrib::Elasticsearch::Quantize.format_url('/my/thing/1/'))
     assert_equal('/my/thing/?/is/cool', Datadog::Contrib::Elasticsearch::Quantize.format_url('/my/thing/1/is/cool'))
     assert_equal('/my/thing/??is=cool', Datadog::Contrib::Elasticsearch::Quantize.format_url('/my/thing/1?is=cool'))
-    assert_equal('/my/thing/1two3/z', Datadog::Contrib::Elasticsearch::Quantize.format_url('/my/thing/1two3/z'))
+    assert_equal('/my/thing/?/z', Datadog::Contrib::Elasticsearch::Quantize.format_url('/my/thing/1two3/z'))
```

Comment:
I'd like to see more test cases that try many more different possible combinations of UUIDs (all letters, some letters, no letters, letters out of range, etc), `_termvector` (present, not present, present with trailing slash, etc), and other edge cases that we found were not working correctly. Right now, I don't trust the tests enough to verify that the bug was properly addressed, or that we're not introducing new ones.